web/src/api/auth/graphql/fetch-user.graphql

```diff
@@ -0,0 +1,7 @@
+query FetchUser($where: user_bool_exp) {
+  user(where: $where) {
```

Comment:
best to restrict to only the potential lookup values (nullifier & auth0Id) ? having this open could be more error prone

Reply:
Done